README.md

```diff
@@ -2,7 +2,7 @@
 
 This is a handbook of coding standards and practices for WordPress that we use at Infinum. WordPress has evolved a lot since its beginnings - from a simple blog CMS to a fully fledge CMS used for a wide range of projects: simple web sites, on line shops, complex multi site projects communicating with a wide range of APIs.
 
-With the increased usage of WordPress in our company, we thought that it would be best to define a set of style guides for projects we work on. We try to follow WordPress Coding Standards for the most part but, being a software developing company working on a wide range of projects, we already use a certain coding standards that differ from the default WordPress Coding Standards.
+With the increased usage of WordPress in our company, we thought that it would be a good idea to define a set of style guides for projects we work on. We try to follow WordPress Coding Standards for the most part but, being a software developing company working on a wide range of projects, we already use a certain coding standards that differ from the default WordPress Coding Standards.
```

Comment:
As the usage of WordPress in our company increased, we thought it would be a good idea to define a set of style guides for the projects we work on. We try to follow the WordPress Coding Standards for the most part, but, being a software development company working on a wide range of projects, we already use certain coding standards that differ from default WordPress Coding Standards.